waku/v2/node/wakunode2.nim

```diff
@@ -138,28 +181,38 @@ proc new*(T: type WakuNode, nodeKey: crypto.PrivateKey,
   let
     rng = crypto.newRng()
     hostAddress = tcpEndPoint(bindIp, bindPort)
+    wsHostAddress = tcpEndPoint(bindIp, wsbindPort) & addWsFlag
     announcedAddresses = if extIp.isNone() or extPort.isNone(): @[]
```

Comment:
The extIp is the tcp Ip, wakunode new function requires change for the new config to work.